app/views/spotlight/sir_trevor/blocks/_quote_block.html.erb

```diff
@@ -1,9 +1,13 @@
 <div class="st__content-block st__content-block--quote">
-  <div class="quote">
-    <div class="quote__content">
-    	<%= sir_trevor_markdown quote_block.text %>
-    </div>
-
+  <blockquote>
```

Comment:
I think there is something odd going on w/ the closing tags here.

It appears that perhaps the `blockquote` is never closed.